uwblueprint/community-fridge-kw
# refactor: migrate to dayjs for create scheduling

## Brief description. What is this change? 

### [Refactor Create Scheduling Endpoint](https://www.notion.so/uwblueprintexecs/Refactor-create-scheduling-endpoint-c60c34df63bc4ed89e14e0f9334d791b)



## Implementation description. How did you make this change?
* Used dayjs library instead of Javascript date when creating schedules
* Fixed the issue where an extra donation after the end date was being added for recurring donations
<img width="769" alt="image" src="https://user-images.githubusercontent.com/37384882/157374264-7c4aa96a-a2de-446a-9268-e075d3cc1376.png">

This still does not fix the issue of daylights saving where donations go an hour ahead. I am not sure how to fix this so please feel free to give any ideas on this.

* Fixed tests


## Steps to test
1. Create daily, weekly, monthly donations and make sure the donations are created with the right dates and an extra donation after the end date is not added like it did previously





## Checklist
